sympy/solvers/ode.py

```diff
@@ -4764,13 +4764,13 @@ def ode_separable_reduced(eq, func, order, match):
     >>> d = f(x).diff(x)
     >>> eq = (x - x**2*f(x))*d - f(x)
     >>> dsolve(eq, hint='separable_reduced')
-    [Eq(f(x), (-sqrt(C1*x**2 + 1) + 1)/x), Eq(f(x), (sqrt(C1*x**2 + 1) + 1)/x)]
+    [Eq(f(x), (1 - sqrt(C1*x**2 + 1))/x), Eq(f(x), (sqrt(C1*x**2 + 1) + 1)/x)]
     >>> pprint(dsolve(eq, hint='separable_reduced'))
-                 ___________                ___________
-                /     2                    /     2
-            - \/  C1*x  + 1  + 1         \/  C1*x  + 1  + 1
-    [f(x) = --------------------, f(x) = ------------------]
-                     x                           x
+                   ___________            ___________
+                  /     2                /     2
+            1 - \/  C1*x  + 1          \/  C1*x  + 1  + 1
```

Comment:
This is one of the only cases where I preferred the previous output. It loses the symmetry between 1+a,1-a by becoming 1-a,a+1.

Reply:
In this case, and many others with roots, the best option would be to have `1+sqrt(x)` and `1-sqrt(x)`. In the light of the comment that you raise below, it's going to be hard to please all of the cases all of the time. As I went through all of the fix-ups for the test cases I did start to wonder whether there is a case for a contextual formatter when formatting sets of results, which aims to present the symmetries, but that's a whole different can of worms.

Still, I think based on all of the patches to the tests that we needed to do this, that on balance this makes things look better a great deal more often than it makes them worse.